docs/source/tagset/elemento-article-title.rst

```diff
@@ -3,49 +3,52 @@
 <article-title>
 ^^^^^^^^^^^^^^^
 
-Aparece em
-  :ref:`elemento-title-group`, 
+Aparece em:
+
+  :ref:`elemento-title-group`
   :ref:`elemento-element-citation`
- 
-Ocorre
+
+Ocorre:
+
   zero ou uma vez
 
+Utilizado para identificar o título do artigo em :ref:`elemento-article-meta`
+ou para especificar um título de documento nas referências em
+:ref:`elemento-element-citation`. Em ambos casos, o atributo ``@xml:lang``
+não deve ser utilizado.
 
-Esta tag pode ser utilizada para especificar o título do artigo em si 
-em :ref:`elemento-article-meta`, ou para especificar um título de documento 
-nas referências em :ref:`elemento-element-citation`. Em ambos os casos, o 
-atributo ``@xml:lang`` não deve ser utilizado.
+.. note:: ``<article-title>`` é obrigatório em :ref:`elemento-title-group`.
 
-.. note:: O elemento ``<article-title>`` é obrigatório em :ref:`elemento-title-group`, .
 
- 
-Exemplo:
- 
+Exemplo 1:
+
 .. code-block:: xml
- 
+
     ...
     <title-group>
         <article-title>The teaching of temporomandibular disorders and  orofacial pain at undergraduate level in Brazilian dental schools</article-title>
     </title-group>
     ...
 
-.. note:: Se o título do artigo ou da referência possuir um subtítulo, ele deve 
-          ser marcado junto a tag ``<article-title>``. Não se deve marcar 
-          nenhum texto separadamente em outras tags 
-          (a mesma regra se aplica a :ref:`elemento-trans-title`).
- 
-Exemplo:
- 
+.. note:: Se o título do artigo ou da referência possuir um subtítulo, este deve
+          ser marcado juntamente com o título sob ``<article-title>``.
+		      Não se deve marcar nenhum título e/ou subtítulo separadamente em outras
```

Comment:
Deve ser. Os editores de texto geralmente permitem vc escolher entre soft-tabs (espaços) ou tab.

-Gustavo

On 28 de junho de 2016 at 09:36:46, Asael C. Silva (notifications@github.com) wrote:
> > ...
> >  
> > The teaching of temporomandibular disorders and orofacial pain  
> at undergraduate level in Brazilian dental schools
> >  
> > ...
> >
> > -.. note:: Se o título do artigo ou da referência possuir um subtítulo, ele deve
> > - ser marcado junto a tag ````. Não se deve marcar
> > - nenhum texto separadamente em outras tags
> > - (a mesma regra se aplica a :ref:`elemento-trans-title`).
> > -
> > -Exemplo:
> > -
> > +.. note:: Se o título do artigo ou da referência possuir um subtítulo, este deve
> > + ser marcado juntamente com o título sob ````.
> > + Não se deve marcar nenhum título e/ou subtítulo separadamente em outras
>  
> Eu acho que é o Atom que você recomendou que está colocando os tabs porque eu não uso tab.  
> Vou tentar achar na configuração do Atom.
>  
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/scieloorg/scielo_publishing_schema/pull/245/files/8e61a9274a0c44867306f3141229e3d3471c5e3f#r68747596